pygenomeworks/genomeworks/cudapoa/cudapoa.pyx

```diff
@@ -162,6 +165,7 @@ cdef class CudaPoaBatch:
             mismatch_score=-6,
             match_score=8,
             cuda_banded_alignment=False,
+            cuda_corrective_alignment=False,
```

Comment:
this can also be updated to use an enum to mirror the C++ side of things

Reply:
@atadkase will take a look at changes related to python binding.

Reply:
awesome, thanks!